lib/Runtime/Library/JavascriptGeneratorFunction.cpp

```diff
@@ -378,8 +378,23 @@ namespace Js
         return false;
     }
 
+    BOOL JavascriptGeneratorFunction::SetAccessors(PropertyId propertyId, Var getter, Var setter, PropertyOperationFlags flags)
+    {
+        if (propertyId == PropertyIds::length)
+        {
+            return this->scriptFunction->SetAccessors(propertyId, getter, setter, flags);
+        }
+
+        return JavascriptFunction::SetAccessors(propertyId, getter, setter, flags);
```

Comment:
JavascriptFunction doesn't override SetAccessors and this call will currently go to DynamicObject::SetAccessors directly.

---
In reply to: [128839022](https://github.com/Microsoft/ChakraCore/pull/3413#discussion_r128839022) [](ancestors = 128839022)